torch/csrc/jit/codegen/cuda/ir_interface_nodes.h

```diff
@@ -301,6 +301,11 @@ class TORCH_CUDA_API TensorView : public Val {
   //
   TensorView* rFactor(const std::vector<int>& axes);
 
+  // Create a new TensorView for all usages of this TensorView.
+  // A common use case is to handle the recompute ComputeAt exception that
+  // occurs when inlining a TensorView used multiple times in a fusion.
+  std::vector<TensorView*> duplicate();
```

Comment:
Please add comments on the constraints about `this` (e.g., having no computeAt view). 

Also, I think it's important to mention here that this duplicates not only `this` but also its origin expression.